lib/utils.py

```diff
@@ -11,9 +12,20 @@ def get_folder(path):
     output_dir.mkdir(parents=True, exist_ok=True)
     return output_dir
 
-def get_image_paths(directory):
-    return [x.path for x in scandir(directory) if
-     any(map(lambda ext: x.name.lower().endswith(ext), image_extensions))]
+def get_image_paths(directory, exclude=[], debug=False):
+    exclude_names = [basename(Path(x).stem[:-1] + Path(x).suffix) for x in exclude]
+    dir_contents = []
+    dir_scanned = list(scandir(directory))
+    for x in dir_scanned:
+        if any(map(lambda ext: x.name.lower().endswith(ext), image_extensions)):
```

Comment:
Personally, I think this is easier to read:
``` python
if any([x.name.lower().endswith(ext) for ext in image_extensions]):
```

Reply:
Me too. That was the original syntax. Changed now.